Make agent's IP table chain names static and detect leftover chain from previous runs (#3249)
metalbear-co/mirrord: changelog.d/3159.changed.md

> The IP table chain names used by the agent are now stable. The agent will detect if another agent is already running or if a previous cleanup failed.

Reviewer: ```suggestion
The IP table chain names used by the agent are no longer randomized. The agent detects if another agent is already running or if a previous cleanup failed.
```
